doozerlib/assembly.py

```diff
@@ -22,6 +22,7 @@ class AssemblyIssueCode(Enum):
     INCONSISTENT_RHCOS_RPMS = 5
     MISSING_INHERITED_DEPENDENCY = 6
     MISSING_RHCOS_CONTAINER = 7
+    INCONSISTENT_OS_COMMIT = 8
```

Comment:
@joepvd I'm not sure if there's a comment explaining the codes somewhere, can we check?

Reply:
hmm i think that does not exist, but it should. That's outside the scope of this PR, though. Stubbed https://issues.redhat.com/browse/ART-5015